src/ResourceManagement/AppService/RuntimeStack.cs

```diff
@@ -15,15 +32,26 @@ public partial class RuntimeStack
         public static readonly RuntimeStack NodeJS_4_4 = new RuntimeStack("NODE", "4.4");
         public static readonly RuntimeStack PHP_5_6 = new RuntimeStack("PHP", "5.6");
         public static readonly RuntimeStack PHP_7_0 = new RuntimeStack("PHP", "7.0");
+        public static readonly RuntimeStack PHP_7_2 = new RuntimeStack("PHP", "7.2");
+        public static readonly RuntimeStack PHP_7_3 = new RuntimeStack("PHP", "7.3");
         public static readonly RuntimeStack NETCore_V1_0 = new RuntimeStack("DOTNETCORE", "1.0");
         public static readonly RuntimeStack NETCore_V1_1 = new RuntimeStack("DOTNETCORE", "1.1");
-        public static readonly RuntimeStack Ruby_2_3 = new RuntimeStack("RUBY", "2.3");
+        public static readonly RuntimeStack NETCore_V2_0 = new RuntimeStack("DOTNETCORE", "2.0");
+        public static readonly RuntimeStack NETCore_V2_1 = new RuntimeStack("DOTNETCORE", "2.1");
+        public static readonly RuntimeStack NETCore_V2_2 = new RuntimeStack("DOTNETCORE", "2.2");
+        public static readonly RuntimeStack Ruby_2_3 = new RuntimeStack("RUBY", "2.3.8");
```

Comment:
We would use major version here. So for RUBY we will have 2.3, 2.4, 2.5, 2.6.
The reason is that minor version might change too quick for us to update in code.

With major version, server would figure out the mirror version.

Reply:
Fixed. Thanks!